probe-rs/targets/STM32L0_Series.yaml

```diff
@@ -4169,13 +4985,13 @@ flash_algorithms:
   flash_properties:
     address_range:
       start: 0x8080000
-      end: 0x8080080
-    page_size: 0x80
+      end: 0x8081800
+    page_size: 0x100
     erased_byte_value: 0x0
     program_page_timeout: 500
     erase_sector_timeout: 500
     sectors:
-    - size: 0x80
+    - size: 0xc00
```

Comment:
these sector sizes look a bit sus, but the old ones also look sus. So I dunno what's correct.

Reply:
me neither but I want to believe that new is better than old

Reply:
Weeeeell there is a not-so-slight chance that this is actually an artifact of flash algo merging. I'm pretty sure I only merge when everything matches except address range, and I don't think we use the sector sizes so there's a good chance this won't cause problems, but we'll see.